### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by jeffpaul**
@dinhtungdu any other tweaks you think could be made to better capture Changelog and Credit info?  Using this repo as a test place for the process of gathering better changelog and prop info, will then move these changes upstream to the `.github` repo.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+### Requirements
+
+Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.  All new code requires documentation and tests to ensure against regressions.
+-->
+
+### Description of the Change
+
+<!--
+We must be able to understand the design of your change from this description. If we can't get a good idea of what the code will be doing from the description here, the pull request may be closed at the maintainers' discretion. Keep in mind that the maintainer reviewing this PR may not be familiar with or have worked with the code here recently, so please walk us through the concepts.  Please include screenshots (if appropriate).
+-->
```
Comment (dinhtungdu) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
### Description of the Change

Fixes #

<!--
We must be able to understand the design of your change from this description. If we can't get a good idea of what the code will be doing from the description here, the pull request may be closed at the maintainers' discretion. Keep in mind that the maintainer reviewing this PR may not be familiar with or have worked with the code here recently, so please walk us through the concepts.  Please include screenshots (if appropriate).
-->
```

**Review by dinhtungdu — COMMENTED** [review]
@jeffpaul This looks good but I think it's a bit overwhelming because of the number of sections.

Personally, I usually skip the Benefits section and include the benefit in the Description of the change.

Another section that can be removed is Applicable Issues. I think we can merge it to the Description of the Change section.

**Comment by jeffpaul** [comment]
@dinhtungdu this reduces the number of sections a bit... any further removal/simplification you think makes sense?

**Review by dinhtungdu — APPROVED** [review]
@jeffpaul this is looking great! 🚢

**Pull request merged**